src/Data/Binary/Put.hs

```diff
@@ -1,6 +1,7 @@
 {-# LANGUAGE CPP #-}
 {-# LANGUAGE FlexibleInstances #-}
-{-# LANGUAGE Safe #-}
+-- {-# LANGUAGE Safe #-}
```

Comment:
Just saw this. Is this module still not Safe? In that case, remove the commented out line.